server/views/review/bassRequest.pug

```diff
@@ -18,25 +18,25 @@ block content
 
     div.pure-g.borderBottomLight
       div.pure-u-1.pure-u-md-1-2
-        h2#bassRequestDetails.heading-medium Proposed BASS area
+        h2#bassRequestDetails.heading-medium Proposed CAS2 area
       div.pure-u-1.pure-u-md-1-2.alignRight
         div.change-details
           if(user.role === 'CA' && stage === 'ELIGIBILITY')
             a#bassEditLink(href="/hdc/bassReferral/bassRequest/change/" + bookingId) Change
 
       div.pure-u-1
         if typeof errorObject.bassReferral === 'string'
-          p.alert.bold(id='proposed-bass-error') Enter a BASS area
+          p.alert.bold(id='proposed-bass-error') Enter a CAS2 area
```

Comment:
Can't quite see how these errors are being triggered here as no user input on this page but wary of removing 😬 so also updated (BASS to CAS2) and left for now